stainless_extraction/src/ty.rs

```diff
@@ -54,11 +54,13 @@ impl<'l, 'tcx> BaseExtractor<'l, 'tcx> {
       TyKind::Int(ast::IntTy::I32) => f.BVType(true, 32).into(),
       TyKind::Int(ast::IntTy::I64) => f.BVType(true, 64).into(),
       TyKind::Int(ast::IntTy::I128) => f.BVType(true, 128).into(),
+      TyKind::Int(ast::IntTy::Isize) => f.BVType(true, 64).into(),
       TyKind::Uint(ast::UintTy::U8) => f.BVType(false, 8).into(),
       TyKind::Uint(ast::UintTy::U16) => f.BVType(false, 16).into(),
       TyKind::Uint(ast::UintTy::U32) => f.BVType(false, 32).into(),
       TyKind::Uint(ast::UintTy::U64) => f.BVType(false, 64).into(),
       TyKind::Uint(ast::UintTy::U128) => f.BVType(false, 128).into(),
+      TyKind::Uint(ast::UintTy::Usize) => f.BVType(false, 64).into(),
```

Comment:
This should map to something like `BVType(true, std::mem::size_of::<isize>() * 8)`.